CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
 
+### Fixed
+
+- exclude acme routes when looking for app routes
```

Comment:
```suggestion
- exclude acme routes when looking for app routes in the blue/green switch
```

Reply:
This part of the code is not used only by the switch task. We also use it in the clean, rollback, deploy.